Eplant/views/ChromosomeViewer/Viewer/GeneInfoPopup.tsx

```diff
@@ -0,0 +1,162 @@
+// -------
+// IMPORTS
+// -------
+
+import React, { FC, useEffect, useState } from 'react'
+import Draggable from 'react-draggable'
+
+import GeneticElement from '@eplant/GeneticElement'
+import arabidopsis from '@eplant/Species/arabidopsis'
+import {
+  useActiveGeneId,
+  useGeneticElements,
+  useSetGeneticElements,
+} from '@eplant/state'
+import CloseIcon from '@mui/icons-material/Close'
+import { Typography } from '@mui/material'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import IconButton from '@mui/material/IconButton'
+import Popover from '@mui/material/Popover'
+import useTheme from '@mui/material/styles/useTheme'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableRow from '@mui/material/TableRow'
+
+import { GeneItem } from '../types'
+
+// TYPES
+interface GeneInfoPopupProps {
+  gene: GeneItem
+  open: boolean
+  anchorOrigin: number[]
+}
+
+//----------
+// COMPONENT
+//----------
+const GeneInfoPopup: FC<GeneInfoPopupProps> = (props) => {
+  const [open, setOpen] = useState(props.open)
+  const [gene, setGene] = useState<GeneItem>(props.gene)
+  // Global State
+  const [activeGeneId, setActiveGeneId] = useActiveGeneId()
+  const geneticElements = useGeneticElements()
```

Comment:
```suggestion
  const [geneticElements, setGeneticElements] = useGeneticElements()
```

All of the Jotai atom hooks work like useStates

Reply:
when i change the geneticelements atom hook to useState format the geneInfoPopup  cannot add genes into the sidebar anymore